src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs

```diff
@@ -65,7 +66,7 @@ public GetAatfNotesRequestHandlerTests()
             A.CallTo(() => systemDataDataAccess.Get()).Returns(systemSettings);
 
             request = new GetAatfNotesRequest(organisation.Id,
-                aatf.Id, fixture.CreateMany<NoteStatus>().ToList(), fixture.Create<string>(), recipientId, wasteType, noteStatus, startDate, endDate);
+                aatf.Id, fixture.CreateMany<NoteStatus>().ToList(), fixture.Create<string>(), recipientId, wasteType, noteStatus, startDate, endDate, 2022, currentDate);
```

Comment:
create private readonly int ComplianceYear property and use 
fixture.Create<int>() to initialise it and pass to the constructor.